Octokit.Tests.Integration/Clients/RepositoryCommentsClientTests.cs

```diff
@@ -377,6 +378,29 @@ public TheCreateMethod()
             Assert.NotNull(retrieved);
         }
 
+        [IntegrationTest]
+        public async Task CanGetReactionPayload()
+        {
+            var commit = await SetupCommitForRepository(_github);
+
+            var comment = new NewCommitComment("test");
+
+            var result = await _github.Repository.Comment.Create(_context.Repository.Id,
+                commit.Sha, comment);
+
+            var reaction = await _reactionsClient.CommitComment.Create(_context.RepositoryOwner, _context.RepositoryName, result.Id, new NewReaction(ReactionType.Confused));
+            var retrieved = await _github.Repository.Comment.Get(_context.RepositoryOwner, _context.RepositoryName, result.Id);
+
+            Assert.True(retrieved.Id > 0);
+            Assert.Equal(1, retrieved.Reactions.TotalCount);
+            Assert.Equal(0, retrieved.Reactions.Plus1);
```

Comment:
I mean have that number of each reaction on an issue then the integration test can retrieve that issue and check each one is desetialzed properly by asserting the unique number